reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -580,6 +586,59 @@ def apply_wiener_filter(projection, ctf, small_number):
         projection_wfilter_f = projection * wfilter
         return projection_wfilter_f
 
+    @staticmethod
+    def compute_ssnr(projections, ctfs_list, small_number=0.01):
```

Comment:
Looks like some good work. Add some more documentation on how your notation corresponds to that in the (Sindelar & Grigorieff, 2011) reference. Reference some equation numbers in (Sindelar & Grigorieff, 2011) in your docs.

* where does `small_number` come in?